### PR TITLE
Add iconset-svg missing dependency. Fixes #327

### LINKED ISSUE #327: Installation is missing required dependency "iron-iconset-svg"
### Description
`bower install vaadin-date-picker` does not install required `iron-iconset-svg`

`vaadin-date-picker-icons.html` contains `<link rel="import" href="../iron-iconset-svg/iron-iconset-svg.html">
`

### Expected outcome
`iron-iconset-svg` is installed and the popup icon is shown

### Actual outcome
404 for `iron-iconset-svg` and no icon in the date picker

### PR DESCRIPTION
**Pull request by manolo**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-date-picker/336)

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
     "iron-media-query": "polymerelements/iron-media-query#^1.0.8",
     "paper-ripple": "polymerelements/paper-ripple#^1.0.5",
     "iron-a11y-keys-behavior": "polymerelements/iron-a11y-keys-behavior#^1.1.1",
-    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#^1.0.5"
+    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#^1.0.5",
+    "iron-iconset-svg": "^1.1.0"
```
Comment (limonte) [inline bower.json]: please be consistent and add full package name before version: `PolymerElements/iron-iconset-svg`

**Comment by manolo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[bower.json, line 36 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336#-KbnPjK-SN5L9JDl1j9o-r1-36:-KbnXwJi7IlS24SHELgE:bb4qc4e) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/ae347411e2449af020cf40008147781fa6a985c4/bower.json#L36)):*
<details><summary><i>Previously, limonte (Limon Monte) wrote…</i></summary><blockquote>

please be consistent and add full package name before version: `PolymerElements/iron-iconset-svg`
</blockquote></details>

Ups, didn't realised, just used the `--save` option in bower.

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336)*

**Comment by limonte** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[bower.json, line 36 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336#-KbnPjK-SN5L9JDl1j9o-r1-36:-KbnYCSdz9OAxw4MP_yS:bsrtv7e) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/ae347411e2449af020cf40008147781fa6a985c4/bower.json#L36)):*
<details><summary><i>Previously, manolo (Manuel Carrasco Moñino) wrote…</i></summary><blockquote>

Ups, didn't realised, just used the `--save` option in bower.

Done.
</blockquote></details>

`PolymerElements/iron-iconset-svg#^1.1.0` not `PolymerElements/^1.1.0`

That's why I'm not happy about those full names. Just version (like `bower install --save` do) will be totally fine.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336)*

**Comment by manolo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[bower.json, line 36 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336#-KbnPjK-SN5L9JDl1j9o-r1-36:-KbnYg6R_-lgbMUKLq7z:b-896fix) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/ae347411e2449af020cf40008147781fa6a985c4/bower.json#L36)):*
<details><summary><i>Previously, limonte (Limon Monte) wrote…</i></summary><blockquote>

`PolymerElements/iron-iconset-svg#^1.1.0` not `PolymerElements/^1.1.0`

That's why I'm not happy about those full names. Just version (like `bower install --save` do) will be totally fine.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336)*

**Comment by limonte** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[bower.json, line 36 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336#-KbnPjK-SN5L9JDl1j9o-r1-36:-KboCj5NzPFoj3BS7dpu:btf86bj) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/ae347411e2449af020cf40008147781fa6a985c4/bower.json#L36)):*
<details><summary><i>Previously, manolo (Manuel Carrasco Moñino) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

still incorrect :) (`#` symbol)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336)*

**Comment by manolo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[bower.json, line 36 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336#-KbnPjK-SN5L9JDl1j9o-r1-36:-KbpgOHQJfedJczxsmf3:b-uy75bh) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/ae347411e2449af020cf40008147781fa6a985c4/bower.json#L36)):*
<details><summary><i>Previously, limonte (Limon Monte) wrote…</i></summary><blockquote>

still incorrect :) (`#` symbol)
</blockquote></details>

Argh, editing with `vi` in an android ssh client is not optimal 

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/336)*

**Pull request merged**